src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/DataElementsCard.tsx

```diff
@@ -66,6 +67,26 @@ const applyDataElementCodes = (code, dataElement) => {
   return updatedDataElement;
 };
 
+const applyNegationRationale = (code, dataElement) => {
+  const modelClass = getDataElementClass(dataElement);
+  const updatedDataElement = new modelClass(dataElement);
```

Comment:
do we need to create a model object again? I believe dataElement is already an instance of one of the cqm models

Reply:
okay, nvm...probably this is to create a new copy